### PR TITLE
DOCS-760: Document Get_image helper function bytes_to_depth_array on Camera API 

### PR DESCRIPTION
**Pull request by sguequierre**
* Document the use case for the RawImage helper function `bytes_to_depth_array`: converting images of `viam.dep` MIME type to a standard image representation 
* Document a code snippet to show conversion to standard image data

NOTE: re. ticket this functionality will eventually be added to all SDKs

### CODE REVIEW

File: docs/components/camera/_index.md [hunk 1]
```diff
@@ -117,8 +117,27 @@ my_camera = Camera.from_robot(robot=robot, name="my_camera")
 frame = await my_cam.get_image()
 ```
 
+<br>
+
+If the `mime_type` of your image is `image/vnd.viam.dep`, pass the returned image data to the Viam Python SDK's [RawImage.bytes_to_depth_array()](https://python.viam.dev/autoapi/viam/media/video/index.html#viam.media.video.RawImage.bytes_to_depth_array) method to decode the raw image data to a standard 2D image representation.
```
Comment (andf-viam) [inline docs/components/camera/_index.md:122]: I would monospace the methodname here

Comment (andf-viam) [inline docs/components/camera/_index.md:122]: I would monospace the method name here:

```suggestion
If the `mime_type` of your image is `image/vnd.viam.dep`, pass the returned image data to the Viam Python SDK's [`RawImage.bytes_to_depth_array()`](https://python.viam.dev/autoapi/viam/media/video/index.html#viam.media.video.RawImage.bytes_to_depth_array) method to decode the raw image data to a standard 2D image representation.
```

File: docs/components/camera/_index.md [hunk 2]
```diff
@@ -117,8 +117,27 @@ my_camera = Camera.from_robot(robot=robot, name="my_camera")
...
+
+For example:
+
+```python {class="line-numbers linkable-line-numbers"}
+# Assume "frame" has a mime_type of "image/vnd.viam.dep"
+frame = await my_cam.get_image()
+
+# Convert "frame" to a standard 2D image representation.
+# Remove the 1st 3x8 bytes and reshape the raw bytes to List[List[Int].
+standard_frame  frame.bytes_to_depth_array()
```
Comment (andf-viam) [inline docs/components/camera/_index.md:132]: one extra space:

```suggestion
standard_frame frame.bytes_to_depth_array()
```

File: docs/components/camera/_index.md [hunk 3]
```diff
@@ -164,7 +183,7 @@ The consumer of this call should decode the bytes into the format suggested by t
 
 **Returns:**
 
-- [(Tuple[bytes,str])](https://docs.python.org/3/library/stdtypes.html#bytes): The pointcloud data as bytes paired with a string representing the mimetype of the pointcloud (for example, PCD).
+- [(Tuple[bytes,str])](https://docs.python.org/3/library/stdtypes.html#bytes): The pointcloud data as bytes paired with a string representing the MIME type of the pointcloud (for example, PCD).
```
Comment (andf-viam) [inline docs/components/camera/_index.md:186]: Good catch!

**Review by andf-viam — APPROVED** [review]
LGTM! Two small nits.

File: docs/components/camera/_index.md [hunk 4]
```diff
@@ -117,8 +117,27 @@ my_camera = Camera.from_robot(robot=robot, name="my_camera")
...
+If the `mime_type` of your image is `image/vnd.viam.dep`, pass the returned image data to the Viam Python SDK's [`RawImage.bytes_to_depth_array()`](https://python.viam.dev/autoapi/viam/media/video/index.html#viam.media.video.RawImage.bytes_to_depth_array) method to decode the raw image data to a standard 2D image representation.
+
+For example:
+
+```python {class="line-numbers linkable-line-numbers"}
+# Assume "frame" has a mime_type of "image/vnd.viam.dep"
+frame = await my_cam.get_image()
+
+# Convert "frame" to a standard 2D image representation.
+# Remove the 1st 3x8 bytes and reshape the raw bytes to List[List[Int].
```
Comment (sguequierre) [inline docs/components/camera/_index.md:131]:
```suggestion
# Remove the 1st 3x8 bytes and reshape the raw bytes to List[List[Int]]].
```

File: docs/components/camera/_index.md [hunk 5]
```diff
@@ -117,8 +117,27 @@ my_camera = Camera.from_robot(robot=robot, name="my_camera")
...
+If the `mime_type` of your image is `image/vnd.viam.dep`, pass the returned image data to the Viam Python SDK's [`RawImage.bytes_to_depth_array()`](https://python.viam.dev/autoapi/viam/media/video/index.html#viam.media.video.RawImage.bytes_to_depth_array) method to decode the raw image data to a standard 2D image representation.
+
+For example:
+
+```python {class="line-numbers linkable-line-numbers"}
+# Assume "frame" has a mime_type of "image/vnd.viam.dep"
+frame = await my_cam.get_image()
+
+# Convert "frame" to a standard 2D image representation.
+# Remove the 1st 3x8 bytes and reshape the raw bytes to List[List[Int]]].
```
Comment (sguequierre) [inline docs/components/camera/_index.md:131]:
```suggestion
# Remove the 1st 3x8 bytes and reshape the raw bytes to List[List[Int]].
```

**Pull request merged**